google-account-plugin/src/com/google/gct/login/ui/GoogleLoginUsersPanel.java

```diff
@@ -56,11 +56,11 @@
   private static final String SIGN_OUT = "Sign Out";
   private static final int MAX_VISIBLE_ROW_COUNT = 3;
 
-  private JBList myList;
-  private DefaultListModel myListModel;
-  private JButton mySignOutButton;
-  private boolean myValueChanged;
-  private boolean myIgnoreSelection;
+  private JBList list;
```

Comment:
I agree. But I want this to be a straight 'my' strip. We can improve naming
in another CL.

On Wed, Mar 9, 2016 at 2:23 PM, Appu Goundan <notifications@github.com>
wrote:

> In
> google-account-plugin/src/com/google/gct/login/ui/GoogleLoginUsersPanel.java
> <https://github.com/GoogleCloudPlatform/gcloud-intellij/pull/490#discussion_r55573195>
> :
>
> > @@ -56,11 +56,11 @@
> >    private static final String SIGN_OUT = "Sign Out";
> >    private static final int MAX_VISIBLE_ROW_COUNT = 3;
> >
> > -  private JBList myList;
> > -  private DefaultListModel myListModel;
> > -  private JButton mySignOutButton;
> > -  private boolean myValueChanged;
> > -  private boolean myIgnoreSelection;
> > +  private JBList list;
>
> not sure I like this name "list"
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/gcloud-intellij/pull/490/files#r55573195>
> .
>